NEWS.md

```diff
@@ -1,13 +1,32 @@
+    -   Life cycles are represented with the `life_cycle` class, which contains a list of `transition` objects. Previously, this data was represented in the `transitions` and `parameters` attributes of a `config`.
+    -   Modified the mechanism for passing functions to transitions to compute transition probability or duration. These functions are now wrapped in a class, `transition_function`. Previously, `config`s contained strings with the name of functions to `get()` from the environment; now we pass the `transition_function`s themselves. This removes the model's dependence on the global R environment, improving reproducibility.
+    -   Using `predictor_spec` and `parameters`, passing parameter and predictor values to `transition_function`s is now done more explicitly.
+-   Updated the example `config`s to conform to the new schema.
+-   Removed several pieces of non-core functionality that we think are better handled by users. This refocuses the package on its core purpose: configuring and running tick population models.
+    -   Removed `read_config()` and `write_config()`, which provided the ability to serialize and deserialize `config`s. These functions were not sufficient to comprehensively serialize all `config`s for reproducible model results. Now, the package only handles in-memory objects.
+    -   Removed `run_all_configs()` and the related `parallel` package dependency. This function was just a thin wrapper over a call to `lapply` or a parallel apply operation.
+    -   Removed `graph_lifecycle()` for visualizing the flow between life stages, and the dependency on the package `igraph`. Instead, `life_cycle` includes a print method with useful information on the flow between life stages.
+    -   Removed `vary_param()` and `vary_many_params()`.
+    -   Removed the graphing functions `graph_population_each_group()` and `graph_population_overall_trend()`. These functions were intended as an easy way to visualize model results, but they did not handle all cases, since model output can be highly variable. We now recommend users write their own code, e.g. with `ggplot2`, to visualize results.
+-   Removed three columns from the data frame output from `run()`: `age_group`, `process` and `infected`. These columns were populated based on the assumption that life stage names were structured as a three-character string like `<process><infected><age_group>`, and used downstream in the graphing functions. Now, users are completely free to name life stages as they wish.
+-   Added `annual_growth_rate()`, for determining the annual factor by which population changes. Results from the `growth_rate()` function can be sensitive to the specific time period being modeled -- this aims to be a more universally applicable alternative.
+-   TODO modified `growth_rate()`
```

Comment:
@dallenmidd we changed `growth_rate()` since the last release. I think it'd be good to describe how the behavior is different. Here's an easy way to see what changed in the code: `git diff 9645414 R/ | grep "[+-]growth_rate <-" -A 8`. Can you help me understand if there's a methodological difference in the code, or if it's just that we changed the implementation with the goal of using base R?